docs/install/install-visual-studio.md

```diff
@@ -93,6 +93,11 @@ To install Visual Studio 2017 RC in a language of your choosing, click the **Lan
 
   > [!IMPORTANT]
   > While Visual Studio 2017 RC in general is supported for use in a production environment, those workloads and components that are marked "Preview" in the installation UI are not supported for use in a production environment.
+  
+## Changing the installer Language
```

Comment:
No need to capitalize "Language" here.  Can just be "Changing the installer language"